api/v1/dspipeline_types.go

```diff
@@ -151,6 +151,12 @@ type APIServer struct {
 	// +kubebuilder:default:=60
 	// +kubebuilder:validation:Optional
 	ArtifactSignedURLExpirySeconds *int `json:"artifactSignedURLExpirySeconds"`
+
+	// The Pipeline Storage type for Pipelines and Pipeline Versions. It can be
+	// either 'database' or 'kubernetes'. Default to 'database'.
```

Comment:
```suggestion
	// The storage for pipeline definitions (pipelines and pipeline versions). It can be
	// either 'database' or 'kubernetes' (Pipeline and PipelineVersion kinds). Defaults to 'database'.
```